src/vos/tests/evt_ctl.c

```diff
@@ -1107,11 +1107,18 @@ test_evt_iter_delete(void **state)
 	int			 sum, expected_sum;
 	struct evt_filter	 filter;
 	uint32_t		 inob;
+	struct	umem_instance		*um_inst;
 
 	rc = evt_create(EVT_FEAT_DEFAULT, 13, arg->ta_uma, arg->ta_root,
 			DAOS_HDL_INVAL, &toh);
 	assert_int_equal(rc, 0);
-
+	um_inst = utest_utx2umm(arg->ta_utx);
+	if (um_inst->umm_id != UMEM_CLASS_VMEM) {
+		rc = utest_init_mem_status();
```

Comment:
init and fini routines go away if you tie them into utest_context initialization and then just pass the context into sync/check routines.   For VMEM, just return 0 initially.

Reply:
Done.